feature/search/src/main/kotlin/uk/govuk/app/search/ui/SearchScreen.kt

```diff
@@ -122,6 +124,7 @@ private fun SearchScreen(
 @Composable
 fun ShowResults(searchResults: List<Result>) {
     val viewModel: SearchViewModel = hiltViewModel()
```

Comment:
Better to pass lambdas to the composables from the route rather than pass the view model or access it directly.

Reply:
Fixed up

Reply:
Fixed up